goon/browserassets/js/browserOutput.js

```diff
@@ -166,30 +166,79 @@ function emojiparse(el) {
 	}
 }
 
-// Colorizes the highlight spans
-function setHighlightColor(match) {
-	match.style.background = opts.highlightColor
+// Recolorizes the highlight spans
+function setHighlightColor() {
+	var highlightspans = document.getElementsByClassName("highlight")
+	for(var i in highlightspans){
+		highlightspans[i].setAttribute("style","background-color:"+opts.highlightColor)
```

Comment:
Let's see, you do it this way because the js interface for modifying existing css rules sucks?

Reply:
Yep, from experience `.style` will occasionally either do nothing, or be undefined even when it IS defined on the element.